test.py

```diff
@@ -5,22 +5,31 @@
 import time
 import lightwood
 
-
 lightwood.config.config.CONFIG.HELPER_MIXERS = False
 mdb = Predictor(name='test_predictor')
-#'rental_price',
-#mdb.learn(to_predict=['neighborhood','rental_price'],from_data="https://mindsdb-example-data.s3.eu-west-2.amazonaws.com/home_rentals.csv",use_gpu=False,stop_training_in_x_seconds=1000, backend='lightwood', unstable_parameters_dict={'use_selfaware_model':True})
-
-p = mdb.predict(when={'number_of_rooms': 3, 'number_of_bathrooms': 2, 'sqft':2411, 'initial_price':3000}, run_confidence_variation_analysis=True, use_gpu=True)
+# 'rental_price',
+mdb.learn(to_predict=['neighborhood', 'rental_price'],
+          from_data="https://mindsdb-example-data.s3.eu-west-2.amazonaws.com/home_rentals.csv",
```

Comment:
This seems to be an indentation issue

Reply:
There is no indentation issue, it runs. Though I accidentally committed this, I used this to test compiling mindsdb and it slipped though. Can revert if neened